pandas/io/clipboard/__init__.py

```diff
@@ -229,6 +220,33 @@ def paste_xsel(primary=False):
     return copy_xsel, paste_xsel
 
 
+def init_wl_clipboard():
+    PRIMARY_SELECTION = "-p"
+
+    def copy_wl(text, primary=False):
+        text = _stringifyText(text)  # Converts non-str values to str.
+        args = ["wl-copy"]
+        if primary:
+            args.append(PRIMARY_SELECTION)
+        if not text:
+            args.append("--clear")
+            subprocess.check_call(args, close_fds=True)
+        else:
+            pass
```

Comment:
`pandas/io/clipboard/__init__.py:235:12: W0107: Unnecessary pass statement (unnecessary-pass)`

Reply:
Sorry about that. Was just about to push a fix ;-).